### ISSUE TITLE
`@redwoodjs/api` failing to include `@babel/runtime-corejs3`

### ISSUE THREAD

**Issue by styfle** [vercel/nft#220]
## Steps to reproduce

```
yarn init -y
yarn add @redwoodjs/api@0.33.0 @redwoodjs/core@0.33.0
echo 'require("@redwoodjs/api/logger")' > index.js
nft build index.js
rm -rf node_modules
node dist/index.js
```

## Error

```
Error: Cannot find module '@babel/runtime-corejs3/core-js/object/define-property'
Require stack:
- example/dist/node_modules/@redwoodjs/api/dist/logger/index.js
- example/dist/node_modules/@redwoodjs/api/logger/index.js
- example/dist/index.js
```

**Comment by styfle** [comment]
Fixed in 0.13.1

**Comment by guybedford** [comment]
@styfle I can't get this to replicate, and it's working fine for me. I've tried a few times can definitely confirm it. Let me know if you can share a direct repo perhaps? Otherwise perhaps double check the versions of nft and that the node_modules is definitely present for the trace?
